--- mate-applets-1.5.0/cpufreq/org.mate.panel.applet.CPUFreqAppletFactory.service.in~	2012-12-08 23:37:32.000000000 +0200
+++ mate-applets-1.5.0/cpufreq/org.mate.panel.applet.CPUFreqAppletFactory.service.in	2012-12-24 23:08:34.722100915 +0200
@@ -1,3 +1,3 @@
 [D-BUS Service]
 Name=org.mate.panel.applet.CPUFreqAppletFactory
-Exec=@LIBEXECDIR@/cpufreq-applet
+Exec=@LIBEXECDIR@/mate-cpufreq-applet
--- mate-applets-1.5.0/cpufreq/org.mate.applets.CPUFreqApplet.mate-panel-applet.in.in~	2012-12-08 23:37:32.000000000 +0200
+++ mate-applets-1.5.0/cpufreq/org.mate.applets.CPUFreqApplet.mate-panel-applet.in.in	2012-12-24 23:09:31.912661587 +0200
@@ -1,6 +1,6 @@
 [Applet Factory]
 Id=CPUFreqAppletFactory
-Location=@LIBEXECDIR@/cpufreq-applet
+Location=@LIBEXECDIR@/mate-cpufreq-applet
 _Name=CPU Frequency Scaling Monitor
 _Description=Monitor the CPU Frequency Scaling
 
--- mate-applets-1.5.0/cpufreq/src/Makefile.am~	2012-12-08 23:37:32.000000000 +0200
+++ mate-applets-1.5.0/cpufreq/src/Makefile.am	2012-12-24 23:11:26.621571494 +0200
@@ -10,13 +10,13 @@
 	$(LIBGLADE_CFLAGS)					\
 	$(DBUS_CFLAGS)
 
-libexec_PROGRAMS = cpufreq-applet
+libexec_PROGRAMS = mate-cpufreq-applet
 
 if HAVE_LIBCPUFREQ
 cpufreq_files=cpufreq-monitor-libcpufreq.c    cpufreq-monitor-libcpufreq.h
 endif
 
-cpufreq_applet_SOURCES =  \
+mate_cpufreq_applet_SOURCES =  \
 	cpufreq-applet.c        	cpufreq-applet.h        	\
 	cpufreq-utils.c			cpufreq-utils.h			\
 	cpufreq-prefs.c         	cpufreq-prefs.h         	\
@@ -29,7 +29,7 @@
 	$(cpufreq_files)						\
 	cpufreq-monitor-cpuinfo.c       cpufreq-monitor-cpuinfo.h
 
-cpufreq_applet_LDADD =  \
+mate_cpufreq_applet_LDADD =  \
 	$(MATE_APPLETS4_LIBS)   	\
 	$(LIBGLADE_LIBS)		\
 	$(LIBCPUFREQ_LIBS)		\
