--- mate-applets-1.10.0/battstat/Makefile.am.orig	2015-05-12 17:58:54.269026483 +0200
+++ mate-applets-1.10.0/battstat/Makefile.am	2015-05-12 20:58:20.245241344 +0200
@@ -89,7 +89,7 @@
 
 CLEANFILES = $(applet_DATA) $(applet_DATA).in $(service_DATA) $(gsettings_SCHEMAS_in)  $(gsettings_SCHEMAS) *.gschema.valid
 
-uidir = $(datadir)/mate/ui
+uidir = $(datadir)/mate-panel/ui
 ui_DATA = battstat-applet-menu.xml
 
 EXTRA_DIST = 							\
--- mate-applets-1.10.0/multiload/Makefile.am~	2013-03-08 02:20:01.000000000 +0200
+++ mate-applets-1.10.0/multiload/Makefile.am	2013-03-14 20:00:28.041998082 +0200
@@ -42,7 +42,7 @@
 	$(gsettings_SCHEMAS_in_in)			\
 	$(ui_DATA)
 
-uidir		= $(datadir)/mate/ui
+uidir		= $(datadir)/mate-panel/ui
 ui_DATA		= multiload-applet-menu.xml
 
 appletdir       = $(datadir)/mate-panel/applets
--- mate-applets-1.10.0/stickynotes/Makefile.am~	2013-03-08 02:20:01.000000000 +0200
+++ mate-applets-1.10.0/stickynotes/Makefile.am	2013-03-14 20:00:40.159222611 +0200
@@ -40,7 +40,7 @@
 
 builder_DATA = $(builder_files)
 
-uidir = $(datadir)/mate/ui
+uidir = $(datadir)/mate-panel/ui
 ui_DATA = $(ui_files)
 
 @INTLTOOL_XML_NOMERGE_RULE@
--- mate-applets-1.10.0/accessx-status/Makefile.am~	2013-03-08 02:20:01.000000000 +0200
+++ mate-applets-1.10.0/accessx-status/Makefile.am	2013-03-14 20:00:51.406407084 +0200
@@ -36,7 +36,7 @@
 		-e "s|\@LIBEXECDIR\@|$(libexecdir)|" \
 		$< > $@
 
-uidir   = $(datadir)/mate/ui
+uidir   = $(datadir)/mate-panel/ui
 ui_DATA = accessx-status-applet-menu.xml
 
 CLEANFILES = $(applet_DATA) $(applet_DATA).in $(service_DATA) $(ACCESSX_STATUS_APPLET_CLEANFILES)
--- mate-applets-1.10.0/trashapplet/Makefile.am~	2013-03-08 02:20:01.000000000 +0200
+++ mate-applets-1.10.0/trashapplet/Makefile.am	2013-03-14 20:01:02.450248862 +0200
@@ -21,7 +21,7 @@
             -e "s|\@LIBEXECDIR\@|$(libexecdir)|" \
             $< > $@
 
-uidir = $(datadir)/mate/ui
+uidir = $(datadir)/mate-panel/ui
 ui_DATA = trashapplet-menu.xml
 
 builder_DATA = trashapplet-empty-progress.ui
--- mate-applets-1.10.0/trashapplet/src/Makefile.am.orig	2015-05-12 21:00:56.068568138 +0200
+++ mate-applets-1.10.0/trashapplet/src/Makefile.am	2015-05-12 21:02:20.331897935 +0200
@@ -1,8 +1,9 @@
+uidir = $(datadir)/mate-panel/ui
 AM_CPPFLAGS = -I$(top_srcdir)		\
 	   $(MATE_APPLETS4_CFLAGS)	\
 	   $(MATEDESKTOP_CFLAGS)	\
 	   $(GIO_CFLAGS)		\
-           -DTRASH_MENU_UI_DIR=\""$(datadir)/mate/ui"\"
+           -DTRASH_MENU_UI_DIR=\""$(uidir)"\"
 
 libexec_PROGRAMS = trashapplet
 
--- mate-applets-1.10.0/cpufreq/Makefile.am~	2013-03-08 02:20:01.000000000 +0200
+++ mate-applets-1.10.0/cpufreq/Makefile.am	2013-03-14 20:02:57.252200866 +0200
@@ -30,7 +30,7 @@
 
 builder_DATA = cpufreq-preferences.ui
 
-uidir = $(datadir)/mate/ui
+uidir = $(datadir)/mate-panel/ui
 ui_DATA = cpufreq-applet-menu.xml
 
 CLEANFILES = \
--- mate-applets-1.10.0/cpufreq/src/Makefile.am.orig	2014-01-01 11:01:50.666615515 +0100
+++ mate-applets-1.10.0/cpufreq/src/Makefile.am	2014-01-01 11:03:13.679945361 +0100
@@ -4,8 +4,9 @@
 
 SUBDIRS = $(selector_SUBDIR)
 
+uidir = $(datadir)/mate-panel/ui
 AM_CPPFLAGS = \
-	-DCPUFREQ_MENU_UI_DIR=\""$(datadir)/mate/ui"\" 	\
+	-DCPUFREQ_MENU_UI_DIR=\""$(uidir)"\" 	\
 	$(MATE_APPLETS4_CFLAGS) 				\
 	$(LIBGLADE_CFLAGS)					\
 	$(DBUS_CFLAGS)
--- mate-applets-1.10.0/mateweather/Makefile.am~	2013-03-08 02:20:01.000000000 +0200
+++ mate-applets-1.10.0/mateweather/Makefile.am	2013-03-14 20:03:47.911199809 +0200
@@ -30,7 +30,7 @@
 	$(NETWORKMANAGER_LIBS)	\
 	$(DBUS_LIBS)
 
-uidir = $(datadir)/mate/ui
+uidir = $(datadir)/mate-panel/ui
 ui_DATA = mateweather-applet-menu.xml
 
 appletdir       = $(datadir)/mate-panel/applets
--- mate-applets-1.10.0/drivemount/Makefile.am~	2013-03-08 02:20:01.000000000 +0200
+++ mate-applets-1.10.0/drivemount/Makefile.am	2013-03-14 20:04:02.671879372 +0200
@@ -20,7 +20,7 @@
 	$(MATE_APPLETS4_LIBS)	\
 	$(MATEDESKTOP_LIBS)
 
-uidir           = $(datadir)/mate/ui
+uidir           = $(datadir)/mate-panel/ui
 ui_DATA         = drivemount-applet-menu.xml
 
 @INTLTOOL_XML_NOMERGE_RULE@
--- mate-applets-1.10.0/geyes/Makefile.am~	2013-03-08 02:20:01.000000000 +0200
+++ mate-applets-1.10.0/geyes/Makefile.am	2013-03-14 20:04:15.615808629 +0200
@@ -40,7 +40,7 @@
             -e "s|\@LIBEXECDIR\@|$(libexecdir)|" \
             $< > $@
 
-uidir = $(datadir)/mate/ui
+uidir = $(datadir)/mate-panel/ui
 ui_DATA = geyes-applet-menu.xml
 
 @INTLTOOL_XML_NOMERGE_RULE@
--- mate-applets-1.10.0/charpick/Makefile.am~	2013-03-08 02:20:01.000000000 +0200
+++ mate-applets-1.10.0/charpick/Makefile.am	2013-03-14 20:04:26.936329764 +0200
@@ -45,7 +45,7 @@
 %.gschema.xml.in: %.gschema.xml.in.in Makefile
 	$(AM_V_GEN) $(SED) -e 's^\@GETTEXT_PACKAGE\@^$(GETTEXT_PACKAGE)^g' < $< > $@
 
-uidir = $(datadir)/mate/ui
+uidir = $(datadir)/mate-panel/ui
 ui_DATA = charpick-applet-menu.xml
 
 CLEANFILES = $(applet_DATA) $(applet_DATA).in $(service_DATA) $(gsettings_SCHEMAS_in)  $(gsettings_SCHEMAS) *.gschema.valid
--- mate-applets-1.14.0/netspeed/data/Makefile.am.old	2016-04-09 21:29:00.894029608 +0200
+++ mate-applets-1.14.0/netspeed/data/Makefile.am	2016-04-09 21:29:14.174108980 +0200
@@ -10,7 +10,7 @@
 		$< > $@
 %.mate-panel-applet: %.mate-panel-applet.in $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*po) ; $(INTLTOOL_MERGE) $(top_srcdir)/po $< $@ -d -u -c $(top_builddir)/po/.intltool-merge-cache
 
-xmluidir   = $(datadir)/mate/ui
+xmluidir   = $(datadir)/mate-panel/ui
 xmlui_DATA = netspeed-menu.xml
 
 servicedir       = $(datadir)/dbus-1/services
